Code/Framework/AzToolsFramework/Tests/FocusMode/EditorFocusModeTests.cpp

```diff
@@ -31,18 +31,58 @@ namespace UnitTest
         EXPECT_EQ(m_focusModeInterface->GetFocusRoot(m_editorEntityContextId), AZ::EntityId());
     }
 
+    TEST_F(EditorFocusModeFixture, GetFocusedEntitiesRoot)
+    {
+        AzToolsFramework::EntityIdList entities = m_focusModeInterface->GetFocusedEntities(m_editorEntityContextId);
+        
+        using ::testing::UnorderedElementsAre;
+        EXPECT_EQ(entities.size(), 6);
+        EXPECT_THAT(
+            entities,
+            UnorderedElementsAre(
+                m_entityMap[CityEntityName],
+                m_entityMap[StreetEntityName],
+                m_entityMap[CarEntityName],
+                m_entityMap[Passenger1EntityName],
+                m_entityMap[SportsCarEntityName],
+                m_entityMap[Passenger2EntityName]
+            )
+        );
+    }
+
+    TEST_F(EditorFocusModeFixture, GetFocusedEntitiesClear)
```

Comment:
nit: Could this be renamed to `GetFocusedEntitiesAfterClear` to make it a bit clearer what it's testing?